### PR TITLE
Request-promise to axios

### PR DESCRIPTION
**Pull request by davidbanham**
Tests currently failing, but create and get both work

### CODE REVIEW

**Comment by davidbanham** [comment]
I think this is now ready to roll.

File: index.js [hunk 1]
```diff
@@ -7,35 +7,55 @@ var request = require('request-promise');
...
 
 Driver.prototype.create = function(data) {
-  return request.post({
-          uri: this.AUTH_URL + '/entities',
-          body: data,
+  data = JSON.parse(JSON.stringify(data));
+  if (!data.emails && data.email) data.emails = [data.email];
+  delete data.email;
+  return axios({
+          url: this.AUTH_URL + '/entities',
```
Comment (nwinch) [inline index.js]: Spacing.

File: index.js [hunk 2]
```diff
@@ -7,35 +7,55 @@ var request = require('request-promise');
...
+  })
+  .then(returnBody);
 };
 
 Driver.prototype.signIn = function(email, password) {
-  return request({
-          uri: this.AUTH_URL + '/login',
-          body:{ email: email, password: password },
+  return axios({
+          url: this.AUTH_URL + '/login',
```
Comment (nwinch) [inline index.js]: Spacing.

File: index.js [hunk 3]
```diff
@@ -7,35 +7,55 @@ var request = require('request-promise');
 * var driver = new Driver('https://username:key@localhost:3000');
 */
 
+var returnBody = res => {
+  return res.data;
+};
 
-Driver = function(AUTH_URL) {
+var Driver = function(AUTH_URL) {
+  var creds = AUTH_URL.split('@')[0].split('//')[1];
```
Comment (nwinch) [inline index.js]: Perhaps check for `AUTH_URL` to be supplied in the correct format so this won't trip up if it's not.

**Comment by nwinch** [comment]
@davidbanham You mentioned when opening this PR the tests currently fail.  I assume they need to be fixed (I haven't run the project to check if so or not!), but other than that and the couple of comments above LGTM :+1:

**Pull request merged**